DEV-3/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace CheckOnFibonacci
+{
+    class Program
+    {
+        // This function checks if the entered number is a positive Fibonacci number
+        // and checks the correctness of the number input
+        static void Main(string[] args)
```

Comment:
Main is entry point, not function for check, Fibonacci number check has to be in a separate method